### ISSUE TITLE
error 'The file '/luv.lua' could not be found'

### ISSUE THREAD

**Issue by aaleksander** [hxdefold/hxdefold#39]
Hi, folks. I took last hxDefold and Haxe 4.3.2. I did every steps in Quick start, but when I build Default project (F5), and receive error: main.lua: The file '/luv.lua' could not be found'. How to correct it?

**Comment by haath** [comment]
Hello, this is a known issue that has to do with the Defold builder, that we cannot exactly "fix" from the Haxe side. The workaround is to create an empty `luv.lua` file at the project root.

An automated fix for this has already been incorporated in [the v2 branch](https://github.com/hxdefold/hxdefold/pull/37). The build macro now generates these empty files at the project root and you don't need to do anything extra.

If you are a new user, I suggest you start using it. You can do this by specifying it as the dependency version in your hxml:

```hxml
--library hxdefold:git:https://github.com/hxdefold/hxdefold.git#v2
```

The documentation for V2 is being written in [the wiki](https://github.com/hxdefold/hxdefold/wiki).

Feel free to report any issues in the V2 pull request 😃

**Comment by haath** [comment]
Closing since we can't fix this on the hxdefold side, and the workaround was just merged into master and relesed with `v2.0.0`.
